anomalib/models/dfkde/normality_model.py

```diff
@@ -20,8 +20,8 @@
 import torch
 from torch import Tensor, nn
 
-from anomalib.core.model.kde import GaussianKDE
-from anomalib.core.model.pca import PCA
+from anomalib.models.components.dimensionality_reduction.pca import PCA
+from anomalib.models.components.stats.kde import GaussianKDE
```

Comment:
This relative import issue could be extended to each component. It would be hard to the user to know the exact location of PCA, GaussianKDE or any other component. Instead we could make the relative import possible so that the user could just do `from anomalib.models import GaussianKDE` or something similar that could easily be remembered

Reply:
We follow a similar structure to scikit learn to organize the models. We could also follow their relative import mechanism